.github/workflows/ros.yml

```diff
@@ -0,0 +1,40 @@
+name: ROS nodes
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  ros1_node:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        release: [melodic, noetic]
+    container: osrf/ros:${{ matrix.release }}-desktop-full
+    steps:
+      - name: Setup cmake
+        uses: jwlawson/actions-setup-cmake@v1.13
+        with:
+          cmake-version: "3.24.x"
```

Comment:
This action was failing on ROS1 because `node` was missing. I also tried to install node with another action but it was also failing, so I basically copy pasted what I had in the gitlab-ci in here to make it work. If you want to try a better solution go ahead, if not let me know so I can merge the latest change ;)

Reply:
Strange, is not working when using `act` but it does work on the real CI. Sorry for the noise